operator/v1/types_network.go

```diff
@@ -119,6 +120,19 @@ type NetworkSpec struct {
 	// migration procedure allows to change the network type and the MTU.
 	// +optional
 	Migration *NetworkMigration `json:"migration,omitempty"`
+
+	// additionalRoutingCapabilities describes components and relevant
+	// configuration providing additional routing capabilities. When set, it
+	// enables such components and the usage of the routing capabilities they
+	// provide for the machine network. Upstream operators, like MetalLB
+	// operator, requiring these capabilities may rely on, or automatically set
+	// this attribute. Network plugins may leverage advanced routing
+	// capabilities acquired through the enablement of these components but may
+	// require specific configuration on their side to do so; refer to their
+	// respective documentation and configuration options.
+	// +openshift:enable:FeatureGate=AdditionalRoutingCapabilities
+	// +optional
+	AdditionalRoutingCapabilities *AdditionalRoutingCapabilities `json:"additionalRoutingCapabilities,omitempty"`
```

Comment:
does it make more sense to call this "AdvancedRouting" instead of AdditionalRoutingCapabilities?

Reply:
We moved away from "Advanced" because that is a very subjective term, the new wording is concrete